cmd/configure_test.go

```diff
@@ -82,6 +80,10 @@ func TestConfigureShow(t *testing.T) {
 }
 
 func TestConfigureToken(t *testing.T) {
+	to := newTerminalOverride()
+	to.override()
+	defer to.reset()
+
```

Comment:
I moved this one to the top since it feels cleaner to keep that setup/teardown consistent.